src/main/scala/tech/cryptonomic/conseil/tezos/ApiOperations.scala

```diff
@@ -403,199 +416,65 @@ object ApiOperations {
     */
   private def fetchSortedAction(order: Option[String], action: Action, sortBy: Option[String]): Action = {
 
+    val sortingField: Option[String] = sortBy.map(_.toLowerCase)
+
+    def sortingOrder[A](col: ColumnOrdered[A]): ColumnOrdered[A] =
```

Comment:
So slick does have some abstraction capability :) Nice find!

Reply:
It does, but I'm afraid it mostly does for internal convenience of the library itself, and those details are hard to find on the docs or any other written source as useful concepts available to the user.

Personally I think this is the worst aspect of the library, it's rather complex under the hood (with all his compile-time abstractions) and there's no easy way to learn how to tweak it as needed, without learning how the code itself works...